### PR TITLE
e2e: added tests for prompts nested folders

### PR DESCRIPTION
**Pull request by irinakartun**
added tests for prompts nested folders

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Deployment status: success

**Comment by irinakartun** [comment]
/deploy-review
>Deployment status: success

File: e2e/src/tests/promptExportImport.test.ts [hunk 1]
```diff
@@ -430,12 +453,303 @@ test('Import file from 1.4 version to prompts and continue working with it', asy
...
+      await folderPrompts
+        .getFolderPrompt(
+          nestedFolders[levelsCount].name,
+          nestedPrompts[levelsCount].name,
+        )
+        .waitFor();
+
+      await folderPrompts
+        .getFolderByName(nestedFolders[levelsCount - 1].name)
+        .waitFor();
```
Comment (IlyaBondar) [inline e2e/src/tests/promptExportImport.test.ts:562]: @irinakartun , why do you use `waitFor()` here and in other places if you already have `await`? does it convert to async-method?

**Review by IlyaBondar — APPROVED** [review]

File: e2e/src/tests/promptExportImport.test.ts (hunk 1 above)
Comment (irinakartun) [reply e2e/src/tests/promptExportImport.test.ts:562]: it simply waits for element visibility on the page

**Pull request merged**